docs/csharp/programming-guide/classes-and-structs/access-modifiers.md

```diff
@@ -16,7 +16,7 @@ All types and type members have an accessibility level. The accessibility level
 - [protected](../../language-reference/keywords/protected.md): The type or member can be accessed only by code in the same `class`, or in a `class` that is derived from that `class`.
 - [internal](../../language-reference/keywords/internal.md): The type or member can be accessed by any code in the same assembly, but not from another assembly.
 - [protected internal](../../language-reference/keywords/protected-internal.md): The type or member can be accessed by any code in the assembly in which it's declared, or from within a derived `class` in another assembly.
-- [private protected](../../language-reference/keywords/private-protected.md): The type or member can be accessed only within its declaring assembly, by code in the same `class` or in a type that is derived from that `class`.
+- [private protected](../../language-reference/keywords/private-protected.md): The type or member can be accessed by types derived from the containing `class`, but only within its containing assembly.
```

Comment:
I think this wording might be clearer.  What do you think:

```suggestion
- [private protected](../../language-reference/keywords/private-protected.md): The type or member can be accessed by types derived from the `class` and declared within its containing assembly.
```

Reply:
@BillWagner Can we make it - 

`The type or member can be accessed by types derived from the `class` that are declared within its containing assembly.`